napari/components/_tests/test_layers_list.py

```diff
@@ -50,6 +50,16 @@ def test_removing_layer():
     assert len(layers) == 0
 
 
+def test_popping_layer():
```

Comment:
That's fair, and i can see why we don't want our tests to proliferate, but I added because we get asked a lot about how to delete layers from the layer list and so wanted to make doubly sure that works as expected